samples/peripheral/lpuart/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _lpuart_sample:
+
+Low Power UART sample
+#####################
+
+The Low Power UART sample demonstrates the capabilities of the :ref:`uart_nrf_lpuart` module.
+
+Overview
+********
+
+The sample implements a simple loopback using a single UART instance.
+The sample has console and logging disabled by default, to demonstrate low power consumption while having UART active.
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF52840DK|
+  * |nRF9160DK|
+
+* The following pins shorted:
+
+  * TX (Arduino Digital Pin 10) with RX (Arduino Digital Pin 11)
+  * Request Pin (Arduino Digital Pin 12) with Response Pin (Arduino Digital Pin 13)
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/peripheral/lpuart`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your board, test it by performing the following steps:
```

Comment:
```suggestion
After programming the sample to your development kit, test it by performing the following steps:
```
We are not using 'board' anymore, use 'development kit'.

Reply:
fixed